specification/authorization/resource-manager/Microsoft.Authorization/preview/2018-05-01-preview/authorization-AccessReviewCalls.json

```diff
@@ -1679,6 +2147,165 @@
       },
       "description": "List of access review decisions."
     },
+    "RoleAssignmentApproval": {
+      "x-ms-azure-resource": true,
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The role assignment approval id."
+        },
+        "name": {
```

Comment:
in path, which param is approvalId? id? Is it supporting `name`? Since it's quite common to input name when querying resource

Reply:
In path, approval id needs to be passed in. The "name" is the same as approval Id

Reply:
To be more clear the `name` property should be the last segment of the value of the `id` property.

---
In reply to: [551552620](https://github.com/Azure/azure-rest-api-specs/pull/12108#discussion_r551552620) [](ancestors = 551552620)